.travis.yml

```diff
@@ -23,13 +23,13 @@ jobs:
     script:
     - make setup/travis
     - operator-sdk build quay.io/aerogear/${OPERATOR_NAME}:$(git rev-parse --short ${TRAVIS_COMMIT})
-  - stage: push
-    script:
-    - export BRANCH=$(if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then echo ${TRAVIS_BRANCH}; else echo ${TRAVIS_PULL_REQUEST_BRANCH}; fi)
-    - export TAG=$(git rev-parse --short ${TRAVIS_COMMIT})
-    - docker login --password "${QUAY_PASSWORD}" --username "${QUAY_USERNAME}" quay.io
-    - make setup/travis
-    - operator-sdk build quay.io/aerogear/${OPERATOR_NAME}:${TAG}
-    - docker push quay.io/aerogear/${OPERATOR_NAME}:${TAG}
-    - docker tag quay.io/aerogear/${OPERATOR_NAME}:${TAG} quay.io/aerogear/${OPERATOR_NAME}:${BRANCH}
-    - docker push quay.io/aerogear/${OPERATOR_NAME}:${BRANCH}
+  # - stage: push
```

Comment:
You can remove this stuff altogether rather than just commenting it out. If we need it back we can always get it from previous commits :)